### DISCUSSION TITLE
提个小建议

### DISCUSSION THREAD

**Discussion by RolixTesk** [share121/inter-knot#4745] [其他]
希望能把电脑端油猴插件的那种设计移植到app上，那样不需要每次发评论都需要跳转到github页面，会方便一点，当然绑定github账号过滤小鬼还是要有的。

**Comment by Emotional-Transparent** [comment]
https://www.lemurbrowser.com/app/zh/?hmsr=wetab
下载一个狐猴浏览器app，这个可以装插件
